packages/manager/src/features/Linodes/LinodesCreate/VPCPanel.test.tsx

```diff
@@ -128,4 +128,68 @@ describe('VPCPanel', () => {
       expect(wrapper.getAllByRole('checkbox')[0]).toBeChecked();
     });
   });
+
+  it('should display an unchecked VPC IPv4 auto-assign checkbox and display the VPC IPv4 input field if there is already a value', async () => {
+    const _props = {
+      ...props,
+      autoassignIPv4WithinVPC: false,
+      region: 'us-east',
+      selectedVPCId: 5,
+      vpcIPv4AddressOfLinode: '10.0.4.3',
+    };
+
+    server.use(
+      rest.get('*/regions', (req, res, ctx) => {
+        const usEast = regionFactory.build({
+          capabilities: ['VPCs'],
+          id: 'us-east',
+        });
+        const regions = regionFactory.buildList(5);
+        return res(ctx.json(makeResourcePage([usEast, ...regions])));
+      })
+    );
+
+    const wrapper = renderWithTheme(<VPCPanel {..._props} />, {
+      flags: { vpc: true },
+      queryClient,
+    });
+
+    await waitFor(() => {
+      // the "Auto-assign a VPC IPv4 address for this Linode in the VPC" checkbox is the first one (0 index)
+      expect(wrapper.getAllByRole('checkbox')[0]).not.toBeChecked();
+
+      wrapper.getByText('VPC IPv4');
+      const vpcIPv4Input = wrapper.getAllByRole('textbox')[2];
+      expect(vpcIPv4Input).toHaveValue('10.0.4.3');
```

Comment:
Selecting the checkbox and input using their label text might make the tests a little more readable and a little less fragile to future changes

```suggestion
      expect(wrapper.getByLabelText('Auto-assign a VPC IPv4 address for this Linode in the VPC')).not.toBeChecked();
      // Using regex here to account for the "(required)" indicator.
      expect(wrapper.getByLabelText(/^VPC IPv4.*/)).toHaveValue('10.0.4.3');
```